css/src/atomics/spacing.scss

```diff
@@ -232,16 +219,13 @@ $layout-sizes: (
 			padding-bottom: sizeValue($sizeKey, $sizeValue) !important;
 		}
 
-		@for $i from 1 through 4 {
-			$class-position: nth($class-positions, $i);
-			$user-position: nth($user-positions, $i);
-
-			.#{$marginKey}#{$separator}#{$class-position}#{$separator}#{$sizeKey}-widescreen {
-				margin-#{$user-position}: sizeValue($sizeKey, $sizeValue) !important;
+		@each $position-key, $direction in $directions {
+			.#{$marginKey}#{$separator}#{$position-key}#{$separator}#{$sizeKey}-widescreen {
+				margin-#{$direction}: sizeValue($sizeKey, $sizeValue) !important;
 			}
 
-			.#{$paddingKey}#{$separator}#{$class-position}#{$separator}#{$sizeKey}-widescreen {
-				padding-#{$user-position}: sizeValue($sizeKey, $sizeValue) !important;
+			.#{$paddingKey}#{$separator}#{$position-key}#{$separator}#{$sizeKey}-widescreen {
+				padding-#{$direction}: sizeValue($sizeKey, $sizeValue) !important;
 			}
 		}
 	}
```

Comment:
Output here is also looking good:


```css
 4.8px 14.4px 0 var(--theme-box-shadow-heavy) !important;
}
.margin-none {
	margin: 0 !important;
}
.padding-none {
	padding: 0 !important;
}
.margin-inline-none {
	margin-right: 0 !important;
	margin-left: 0 !important;
}
.padding-inline-none {
	padding-right: 0 !important;
	padding-left: 0 !important;
}
.margin-block-none {
	margin-top: 0 !important;
	margin-bottom: 0 !important;
}
.padding-block-none {
	padding-top: 0 !important;
	padding-bottom: 0 !important;
}
.margin-top-none {
	margin-top: 0 !important;
}
.padding-top-none {
	padding-top: 0 !important;
}
.margin-right-none {
	margin-right: 0 !important;
}
.padding-right-none {
	padding-right: 0 !important;
}
.margin-bottom-none {
	margin-bottom: 0 !important;
}
.padding-bottom-none {
	padding-bottom: 0 !important;
}
.margin-left-none {
	margin-left: 0 !important;
}
.padding-left-none {
	padding-left: 0 !important;
}
.margin-xs {
	margin: 1rem !important;
}
.padding-xs {
	padding: 1rem !important;
}
.margin-inline-xs {
	margin-right: 1rem !important;
	margin-left: 1rem !important;
}
.padding-inline-xs {
	padding-right: 1rem !important;
	padding-left: 1rem !important;
}
.margin-block-xs {
	margin-top: 1rem !important;
	margin-bottom: 1rem !important;
}
.padding-block-xs {
	padding-top: 1rem !important;
	padding-bottom: 1rem !important;
}
.margin-top-xs {
	margin-top: 1rem !important;
}
.padding-top-xs {
	padding-top: 1rem !important;
}
.margin-right-xs {
	margin-right: 1rem !important;
}
.padding-right-xs {
	padding-right: 1rem !important;
}
.margin-bottom-xs {
	margin-bottom: 1rem !important;
}
.padding-bottom-xs {
	padding-bottom: 1rem !important;
}
.margin-left-xs {
	margin-left: 1rem !important;
}
.padding-left-xs {
	padding-left: 1rem !important;
}
.margin-s {
	margin: 1.5rem !important;
}
.padding-s {
	padding: 1.5rem !important;
}
.margin-inline-s {
	margin-right: 1.5rem !important;
	margin-left: 1.5rem !important;
}
.padding-inline-s {
	padding-right: 1.5rem !important;
	padding-left: 1.5rem !important;
}
.margin-block-s {
	margin-top: 1.5rem !important;
	margin-bottom: 1.5rem !important;
}
.padding-block-s {
	padding-top: 1.5rem !important;
	padding-bottom: 1.5rem !important;
}
.margin-top-s {
	margin-top: 1.5rem !important;
}
.padding-top-s {
	padding-top: 1.5rem !important;
}
.margin-right-s {
	margin-right: 1.5rem !important;
}
.padding-right-s {
	padding-right: 1.5rem !important;
}
.margin-bottom-s {
	margin-bottom: 1.5rem !important;
}
.padding-bottom-s {
	padding-bottom: 1.5rem !important;
}
.margin-left-s {
	margin-left: 1.5rem !important;
}
.padding-left-s {
	padding-left: 1.5rem !important;
}
.margin-m {
	margin: 2rem !important;
}
.padding-m {
	padding: 2rem !important;
}
.margin-inline-m {
	margin-right: 2rem !important;
	margin-left: 2rem !important;
}
.padding-inline-m {
	padding-right: 2rem !important;
	padding-left: 2rem !important;
}
.margin-block-m {
	margin-top: 2rem !important;
	margin-bottom: 2rem !important;
}
.padding-block-m {
	padding-top: 2rem !important;
	padding-bottom: 2rem !important;
}
.margin-top-m {
	margin-top: 2rem !important;
}
.padding-top-m {
	padding-top: 2rem !important;
}
.margin-right-m {
	margin-right: 2rem !important;
}
.padding-right-m {
	padding-right: 2rem !important;
}
.margin-bottom-m {
	margin-bottom: 2rem !important;
}
.padding-bottom-m {
	padding-bottom: 2rem !important;
}
.margin-left-m {
	margin-left: 2rem !important;
}
.padding-left-m {
	padding-left: 2rem !important;
}
.margin-l {
	margin: 3rem !important;
}
.padding-l {
	padding: 3rem !important;
}
.margin-inline-l {
	margin-right: 3rem !important;
	margin-left: 3rem !important;
}
.padding-inline-l {
	padding-right: 3rem !important;
	padding-left: 3rem !important;
}
.margin-block-l {
	margin-top: 3rem !important;
	margin-bottom: 3rem !important;
}
.padding-block-l {
	padding-top: 3rem !important;
	padding-bottom: 3rem !important;
}
.margin-top-l {
	margin-top: 3rem !important;
}
.padding-top-l {
	padding-top: 3rem !important;
}
.margin-right-l {
	margin-right: 3rem !important;
}
.padding-right-l {
	padding-right: 3rem !important;
}
.margin-bottom-l {
	margin-bottom: 3rem !important;
}
.padding-bottom-l {
	padding-bottom: 3rem !important;
}
.margin-left-l {
	margin-left: 3rem !important;
}
.padding-left-l {
	padding-left: 3rem !important;
}
.margin-xl {
	margin: 4rem !important;
}
.padding-xl {
	padding: 4rem !important;
}
.margin-inline-xl {
	margin-right: 4rem !important;
	margin-left: 4rem !important;
}
.padding-inline-xl {
	padding-right: 4rem !important;
	padding-left: 4rem !important;
}
.margin-block-xl {
	margin-top: 4rem !important;
	margin-bottom: 4rem !important;
}
.padding-block-xl {
	padding-top: 4rem !important;
	padding-bottom: 4rem !important;
}
.margin-top-xl {
	margin-top: 4rem !important;
}
.padding-top-xl {
	padding-top: 4rem !important;
}
.margin-right-xl {
	margin-right: 4rem !important;
}
.padding-right-xl {
	padding-right: 4rem !important;
}
.margin-bottom-xl {
	margin-bottom: 4rem !important;
}
.padding-bottom-xl {
	padding-bottom: 4rem !important;
}
.margin-left-xl {
	margin-left: 4rem !important;
}
.padding-left-xl {
	padding-left: 4rem !important;
}
.margin-xxl {
	margin: 6rem !important;
}
.padding-xxl {
	padding: 6rem !important;
}
.margin-inline-xxl {
	margin-right: 6rem !important;
	margin-left: 6rem !important;
}
.padding-inline-xxl {
	padding-right: 6rem !important;
	padding-left: 6rem !important;
}
.margin-block-xxl {
	margin-top: 6rem !important;
	margin-bottom: 6rem !important;
}
.padding-block-xxl {
	padding-top: 6rem !important;
	padding-bottom: 6rem !important;
}
.margin-top-xxl {
	margin-top: 6rem !important;
}
.padding-top-xxl {
	padding-top: 6rem !important;
}
.margin-right-xxl {
	margin-right: 6rem !important;
}
.padding-right-xxl {
	padding-right: 6rem !important;
}
.margin-bottom-xxl {
	margin-bottom: 6rem !important;
}
.padding-bottom-xxl {
	padding-bottom: 6rem !important;
}
.margin-left-xxl {
	margin-left: 6rem !important;
}
.padding-left-xxl {
	padding-left: 6rem !important;
}
.margin-xxxl {
	margin: 8rem !important;
}
.padding-xxxl {
	padding: 8rem !important;
}
.margin-inline-xxxl {
	margin-right: 8rem !important;
	margin-left: 8rem !important;
}
.padding-inline-xxxl {
	padding-right: 8rem !important;
	padding-left: 8rem !important;
}
.margin-block-xxxl {
	margin-top: 8rem !important;
	margin-bottom: 8rem !important;
}
.padding-block-xxxl {
	padding-top: 8rem !important;
	padding-bottom: 8rem !important;
}
.margin-top-xxxl {
	margin-top: 8rem !important;
}
.padding-top-xxxl {
	padding-top: 8rem !important;
}
.margin-right-xxxl {
	margin-right: 8rem !important;
}
.padding-right-xxxl {
	padding-right: 8rem !important;
}
.margin-bottom-xxxl {
	margin-bottom: 8rem !important;
}
.padding-bottom-xxxl {
	padding-bottom: 8rem !important;
}
.margin-left-xxxl {
	margin-left: 8rem !important;
}
.padding-left-xxxl {
	padding-left: 8rem !important;
}
@media print, screen and (min-width: 768px) {
	.margin-none-tablet {
		margin: 0 !important;
	}
	.padding-none-tablet {
		padding: 0 !important;
	}
	.margin-inline-none-tablet {
		margin-right: 0 !important;
		margin-left: 0 !important;
	}
	.padding-inline-none-tablet {
		padding-right: 0 !important;
		padding-left: 0 !important;
	}
	.margin-block-none-tablet {
		margin-top: 0 !important;
		margin-bottom: 0 !important;
	}
	.padding-block-none-tablet {
		padding-top: 0 !important;
		padding-bottom: 0 !important;
	}
	.margin-top-none-tablet {
		margin-top: 0 !important;
	}
	.padding-top-none-tablet {
		padding-top: 0 !important;
	}
	.margin-right-none-tablet {
		margin-right: 0 !important;
	}
	.padding-right-none-tablet {
		padding-right: 0 !important;
	}
	.margin-bottom-none-tablet {
		margin-bottom: 0 !important;
	}
	.padding-bottom-none-tablet {
		padding-bottom: 0 !important;
	}
	.margin-left-none-tablet {
		margin-left: 0 !important;
	}
	.padding-left-none-tablet {
		padding-left: 0 !important;
	}
	.margin-xs-tablet {
		margin: 1rem !important;
	}
	.padding-xs-tablet {
		padding: 1rem !important;
	}
	.margin-inline-xs-tablet {
		margin-right: 1rem !important;
		margin-left: 1rem !important;
	}
	.padding-inline-xs-tablet {
		padding-right: 1rem !important;
		padding-left: 1rem !important;
	}
	.margin-block-xs-tablet {
		margin-top: 1rem !important;
		margin-bottom: 1rem !important;
	}
	.padding-block-xs-tablet {
		padding-top: 1rem !important;
		padding-bottom: 1rem !important;
	}
	.margin-top-xs-tablet {
		margin-top: 1rem !important;
	}
	.padding-top-xs-tablet {
		padding-top: 1rem !important;
	}
	.margin-right-xs-tablet {
		margin-right: 1rem !important;
	}
	.padding-right-xs-tablet {
		padding-right: 1rem !important;
	}
	.margin-bottom-xs-tablet {
		margin-bottom: 1rem !important;
	}
	.padding-bottom-xs-tablet {
		padding-bottom: 1rem !important;
	}
	.margin-left-xs-tablet {
		margin-left: 1rem !important;
	}
	.padding-left-xs-tablet {
		padding-left: 1rem !important;
	}
	.margin-s-tablet {
		margin: 1.5rem !important;
	}
	.padding-s-tablet {
		padding: 1.5rem !important;
	}
	.margin-inline-s-tablet {
		margin-right: 1.5rem !important;
		margin-left: 1.5rem !important;
	}
	.padding-inline-s-tablet {
		padding-right: 1.5rem !important;
		padding-left: 1.5rem !important;
	}
	.margin-block-s-tablet {
		margin-top: 1.5rem !important;
		margin-bottom: 1.5rem !important;
	}
	.padding-block-s-tablet {
		padding-top: 1.5rem !important;
		padding-bottom: 1.5rem !important;
	}
	.margin-top-s-tablet {
		margin-top: 1.5rem !important;
	}
	.padding-top-s-tablet {
		padding-top: 1.5rem !important;
	}
	.margin-right-s-tablet {
		margin-right: 1.5rem !important;
	}
	.padding-right-s-tablet {
		padding-right: 1.5rem !important;
	}
	.margin-bottom-s-tablet {
		margin-bottom: 1.5rem !important;
	}
	.padding-bottom-s-tablet {
		padding-bottom: 1.5rem !important;
	}
	.margin-left-s-tablet {
		margin-left: 1.5rem !important;
	}
	.padding-left-s-tablet {
		padding-left: 1.5rem !important;
	}
	.margin-m-tablet {
		margin: 2rem !important;
	}
	.padding-m-tablet {
		padding: 2rem !important;
	}
	.margin-inline-m-tablet {
		margin-right: 2rem !important;
		margin-left: 2rem !important;
	}
	.padding-inline-m-tablet {
		padding-right: 2rem !important;
		padding-left: 2rem !important;
	}
	.margin-block-m-tablet {
		margin-top: 2rem !important;
		margin-bottom: 2rem !important;
	}
	.padding-block-m-tablet {
		padding-top: 2rem !important;
		padding-bottom: 2rem !important;
	}
	.margin-top-m-tablet {
		margin-top: 2rem !important;
	}
	.padding-top-m-tablet {
		padding-top: 2rem !important;
	}
	.margin-right-m-tablet {
		margin-right: 2rem !important;
	}
	.padding-right-m-tablet {
		padding-right: 2rem !important;
	}
	.margin-bottom-m-tablet {
		margin-bottom: 2rem !important;
	}
	.padding-bottom-m-tablet {
		padding-bottom: 2rem !important;
	}
	.margin-left-m-tablet {
		margin-left: 2rem !important;
	}
	.padding-left-m-tablet {
		padding-left: 2rem !important;
	}
	.margin-l-tablet {
		margin: 3rem !important;
	}
	.padding-l-tablet {
		padding: 3rem !important;
	}
	.margin-inline-l-tablet {
		margin-right: 3rem !important;
		margin-left: 3rem !important;
	}
	.padding-inline-l-tablet {
		padding-right: 3rem !important;
		padding-left: 3rem !important;
	}
	.margin-block-l-tablet {
		margin-top: 3rem !important;
		margin-bottom: 3rem !important;
	}
	.padding-block-l-tablet {
		padding-top: 3rem !important;
		padding-bottom: 3rem !important;
	}
	.margin-top-l-tablet {
		margin-top: 3rem !important;
	}
	.padding-top-l-tablet {
		padding-top: 3rem !important;
	}
	.margin-right-l-tablet {
		margin-right: 3rem !important;
	}
	.padding-right-l-tablet {
		padding-right: 3rem !important;
	}
	.margin-bottom-l-tablet {
		margin-bottom: 3rem !important;
	}
	.padding-bottom-l-tablet {
		padding-bottom: 3rem !important;
	}
	.margin-left-l-tablet {
		margin-left: 3rem !important;
	}
	.padding-left-l-tablet {
		padding-left: 3rem !important;
	}
	.margin-xl-tablet {
		margin: 4rem !important;
	}
	.padding-xl-tablet {
		padding: 4rem !important;
	}
	.margin-inline-xl-tablet {
		margin-right: 4rem !important;
		margin-left: 4rem !important;
	}
	.padding-inline-xl-tablet {
		padding-right: 4rem !important;
		padding-left: 4rem !important;
	}
	.margin-block-xl-tablet {
		margin-top: 4rem !important;
		margin-bottom: 4rem !important;
	}
	.padding-block-xl-tablet {
		padding-top: 4rem !important;
		padding-bottom: 4rem !important;
	}
	.margin-top-xl-tablet {
		margin-top: 4rem !important;
	}
	.padding-top-xl-tablet {
		padding-top: 4rem !important;
	}
	.margin-right-xl-tablet {
		margin-right: 4rem !important;
	}
	.padding-right-xl-tablet {
		padding-right: 4rem !important;
	}
	.margin-bottom-xl-tablet {
		margin-bottom: 4rem !important;
	}
	.padding-bottom-xl-tablet {
		padding-bottom: 4rem !important;
	}
	.margin-left-xl-tablet {
		margin-left: 4rem !important;
	}
	.padding-left-xl-tablet {
		padding-left: 4rem !important;
	}
	.margin-xxl-tablet {
		margin: 6rem !important;
	}
	.padding-xxl-tablet {
		padding: 6rem !important;
	}
	.margin-inline-xxl-tablet {
		margin-right: 6rem !important;
		margin-left: 6rem !important;
	}
	.padding-inline-xxl-tablet {
		padding-right: 6rem !important;
		padding-left: 6rem !important;
	}
	.margin-block-xxl-tablet {
		margin-top: 6rem !important;
		margin-bottom: 6rem !important;
	}
	.padding-block-xxl-tablet {
		padding-top: 6rem !important;
		padding-bottom: 6rem !important;
	}
	.margin-top-xxl-tablet {
		margin-top: 6rem !important;
	}
	.padding-top-xxl-tablet {
		padding-top: 6rem !important;
	}
	.margin-right-xxl-tablet {
		margin-right: 6rem !important;
	}
	.padding-right-xxl-tablet {
		padding-right: 6rem !important;
	}
	.margin-bottom-xxl-tablet {
		margin-bottom: 6rem !important;
	}
	.padding-bottom-xxl-tablet {
		padding-bottom: 6rem !important;
	}
	.margin-left-xxl-tablet {
		margin-left: 6rem !important;
	}
	.padding-left-xxl-tablet {
		padding-left: 6rem !important;
	}
	.margin-xxxl-tablet {
		margin: 8rem !important;
	}
	.padding-xxxl-tablet {
		padding: 8rem !important;
	}
	.margin-inline-xxxl-tablet {
		margin-right: 8rem !important;
		margin-left: 8rem !important;
	}
	.padding-inline-xxxl-tablet {
		padding-right: 8rem !important;
		padding-left: 8rem !important;
	}
	.margin-block-xxxl-tablet {
		margin-top: 8rem !important;
		margin-bottom: 8rem !important;
	}
	.padding-block-xxxl-tablet {
		padding-top: 8rem !important;
		padding-bottom: 8rem !important;
	}
	.margin-top-xxxl-tablet {
		margin-top: 8rem !important;
	}
	.padding-top-xxxl-tablet {
		padding-top: 8rem !important;
	}
	.margin-right-xxxl-tablet {
		margin-right: 8rem !important;
	}
	.padding-right-xxxl-tablet {
		padding-right: 8rem !important;
	}
	.margin-bottom-xxxl-tablet {
		margin-bottom: 8rem !important;
	}
	.padding-bottom-xxxl-tablet {
		padding-bottom: 8rem !important;
	}
	.margin-left-xxxl-tablet {
		margin-left: 8rem !important;
	}
	.padding-left-xxxl-tablet {
		padding-left: 8rem !important;
	}
}
@media screen and (min-width: 1088px) {
	.margin-none-desktop {
		margin: 0 !important;
	}
	.padding-none-desktop {
		padding: 0 !important;
	}
	.margin-inline-none-desktop {
		margin-right: 0 !important;
		margin-left: 0 !important;
	}
	.padding-inline-none-desktop {
		padding-right: 0 !important;
		padding-left: 0 !important;
	}
	.margin-block-none-desktop {
		margin-top: 0 !important;
		margin-bottom: 0 !important;
	}
	.padding-block-none-desktop {
		padding-top: 0 !important;
		padding-bottom: 0 !important;
	}
	.margin-top-none-desktop {
		margin-top: 0 !important;
	}
	.padding-top-none-desktop {
		padding-top: 0 !important;
	}
	.margin-right-none-desktop {
		margin-right: 0 !important;
	}
	.padding-right-none-desktop {
		padding-right: 0 !important;
	}
	.margin-bottom-none-desktop {
		margin-bottom: 0 !important;
	}
	.padding-bottom-none-desktop {
		padding-bottom: 0 !important;
	}
	.margin-left-none-desktop {
		margin-left: 0 !important;
	}
	.padding-left-none-desktop {
		padding-left: 0 !important;
	}
	.margin-xs-desktop {
		margin: 1rem !important;
	}
	.padding-xs-desktop {
		padding: 1rem !important;
	}
	.margin-inline-xs-desktop {
		margin-right: 1rem !important;
		margin-left: 1rem !important;
	}
	.padding-inline-xs-desktop {
		padding-right: 1rem !important;
		padding-left: 1rem !important;
	}
	.margin-block-xs-desktop {
		margin-top: 1rem !important;
		margin-bottom: 1rem !important;
	}
	.padding-block-xs-desktop {
		padding-top: 1rem !important;
		padding-bottom: 1rem !important;
	}
	.margin-top-xs-desktop {
		margin-top: 1rem !important;
	}
	.padding-top-xs-desktop {
		padding-top: 1rem !important;
	}
	.margin-right-xs-desktop {
		margin-right: 1rem !important;
	}
	.padding-right-xs-desktop {
		padding-right: 1rem !important;
	}
	.margin-bottom-xs-desktop {
		margin-bottom: 1rem !important;
	}
	.padding-bottom-xs-desktop {
		padding-bottom: 1rem !important;
	}
	.margin-left-xs-desktop {
		margin-left: 1rem !important;
	}
	.padding-left-xs-desktop {
		padding-left: 1rem !important;
	}
	.margin-s-desktop {
		margin: 1.5rem !important;
	}
	.padding-s-desktop {
		padding: 1.5rem !important;
	}
	.margin-inline-s-desktop {
		margin-right: 1.5rem !important;
		margin-left: 1.5rem !important;
	}
	.padding-inline-s-desktop {
		padding-right: 1.5rem !important;
		padding-left: 1.5rem !important;
	}
	.margin-block-s-desktop {
		margin-top: 1.5rem !important;
		margin-bottom: 1.5rem !important;
	}
	.padding-block-s-desktop {
		padding-top: 1.5rem !important;
		padding-bottom: 1.5rem !important;
	}
	.margin-top-s-desktop {
		margin-top: 1.5rem !important;
	}
	.padding-top-s-desktop {
		padding-top: 1.5rem !important;
	}
	.margin-right-s-desktop {
		margin-right: 1.5rem !important;
	}
	.padding-right-s-desktop {
		padding-right: 1.5rem !important;
	}
	.margin-bottom-s-desktop {
		margin-bottom: 1.5rem !important;
	}
	.padding-bottom-s-desktop {
		padding-bottom: 1.5rem !important;
	}
	.margin-left-s-desktop {
		margin-left: 1.5rem !important;
	}
	.padding-left-s-desktop {
		padding-left: 1.5rem !important;
	}
	.margin-m-desktop {
		margin: 2rem !important;
	}
	.padding-m-desktop {
		padding: 2rem !important;
	}
	.margin-inline-m-desktop {
		margin-right: 2rem !important;
		margin-left: 2rem !important;
	}
	.padding-inline-m-desktop {
		padding-right: 2rem !important;
		padding-left: 2rem !important;
	}
	.margin-block-m-desktop {
		margin-top: 2rem !important;
		margin-bottom: 2rem !important;
	}
	.padding-block-m-desktop {
		padding-top: 2rem !important;
		padding-bottom: 2rem !important;
	}
	.margin-top-m-desktop {
		margin-top: 2rem !important;
	}
	.padding-top-m-desktop {
		padding-top: 2rem !important;
	}
	.margin-right-m-desktop {
		margin-right: 2rem !important;
	}
	.padding-right-m-desktop {
		padding-right: 2rem !important;
	}
	.margin-bottom-m-desktop {
		margin-bottom: 2rem !important;
	}
	.padding-bottom-m-desktop {
		padding-bottom: 2rem !important;
	}
	.margin-left-m-desktop {
		margin-left: 2rem !important;
	}
	.padding-left-m-desktop {
		padding-left: 2rem !important;
	}
	.margin-l-desktop {
		margin: 3rem !important;
	}
	.padding-l-desktop {
		padding: 3rem !important;
	}
	.margin-inline-l-desktop {
		margin-right: 3rem !important;
		margin-left: 3rem !important;
	}
	.padding-inline-l-desktop {
		padding-right: 3rem !important;
		padding-left: 3rem !important;
	}
	.margin-block-l-desktop {
		margin-top: 3rem !important;
		margin-bottom: 3rem !important;
	}
	.padding-block-l-desktop {
		padding-top: 3rem !important;
		padding-bottom: 3rem !important;
	}
	.margin-top-l-desktop {
		margin-top: 3rem !important;
	}
	.padding-top-l-desktop {
		padding-top: 3rem !important;
	}
	.margin-right-l-desktop {
		margin-right: 3rem !important;
	}
	.padding-right-l-desktop {
		padding-right: 3rem !important;
	}
	.margin-bottom-l-desktop {
		margin-bottom: 3rem !important;
	}
	.padding-bottom-l-desktop {
		padding-bottom: 3rem !important;
	}
	.margin-left-l-desktop {
		margin-left: 3rem !important;
	}
	.padding-left-l-desktop {
		padding-left: 3rem !important;
	}
	.margin-xl-desktop {
		margin: 4rem !important;
	}
	.padding-xl-desktop {
		padding: 4rem !important;
	}
	.margin-inline-xl-desktop {
		margin-right: 4rem !important;
		margin-left: 4rem !important;
	}
	.padding-inline-xl-desktop {
		padding-right: 4rem !important;
		padding-left: 4rem !important;
	}
	.margin-block-xl-desktop {
		margin-top: 4rem !important;
		margin-bottom: 4rem !important;
	}
	.padding-block-xl-desktop {
		padding-top: 4rem !important;
		padding-bottom: 4rem !important;
	}
	.margin-top-xl-desktop {
		margin-top: 4rem !important;
	}
	.padding-top-xl-desktop {
		padding-top: 4rem !important;
	}
	.margin-right-xl-desktop {
		margin-right: 4rem !important;
	}
	.padding-right-xl-desktop {
		padding-right: 4rem !important;
	}
	.margin-bottom-xl-desktop {
		margin-bottom: 4rem !important;
	}
	.padding-bottom-xl-desktop {
		padding-bottom: 4rem !important;
	}
	.margin-left-xl-desktop {
		margin-left: 4rem !important;
	}
	.padding-left-xl-desktop {
		padding-left: 4rem !important;
	}
	.margin-xxl-desktop {
		margin: 6rem !important;
	}
	.padding-xxl-desktop {
		padding: 6rem !important;
	}
	.margin-inline-xxl-desktop {
		margin-right: 6rem !important;
		margin-left: 6rem !important;
	}
	.padding-inline-xxl-desktop {
		padding-right: 6rem !important;
		padding-left: 6rem !important;
	}
	.margin-block-xxl-desktop {
		margin-top: 6rem !important;
		margin-bottom: 6rem !important;
	}
	.padding-block-xxl-desktop {
		padding-top: 6rem !important;
		padding-bottom: 6rem !important;
	}
	.margin-top-xxl-desktop {
		margin-top: 6rem !important;
	}
	.padding-top-xxl-desktop {
		padding-top: 6rem !important;
	}
	.margin-right-xxl-desktop {
		margin-right: 6rem !important;
	}
	.padding-right-xxl-desktop {
		padding-right: 6rem !important;
	}
	.margin-bottom-xxl-desktop {
		margin-bottom: 6rem !important;
	}
	.padding-bottom-xxl-desktop {
		padding-bottom: 6rem !important;
	}
	.margin-left-xxl-desktop {
		margin-left: 6rem !important;
	}
	.padding-left-xxl-desktop {
		padding-left: 6rem !important;
	}
	.margin-xxxl-desktop {
		margin: 8rem !important;
	}
	.padding-xxxl-desktop {
		padding: 8rem !important;
	}
	.margin-inline-xxxl-desktop {
		margin-right: 8rem !important;
		margin-left: 8rem !important;
	}
	.padding-inline-xxxl-desktop {
		padding-right: 8rem !important;
		padding-left: 8rem !important;
	}
	.margin-block-xxxl-desktop {
		margin-top: 8rem !important;
		margin-bottom: 8rem !important;
	}
	.padding-block-xxxl-desktop {
		padding-top: 8rem !important;
		padding-bottom: 8rem !important;
	}
	.margin-top-xxxl-desktop {
		margin-top: 8rem !important;
	}
	.padding-top-xxxl-desktop {
		padding-top: 8rem !important;
	}
	.margin-right-xxxl-desktop {
		margin-right: 8rem !important;
	}
	.padding-right-xxxl-desktop {
		padding-right: 8rem !important;
	}
	.margin-bottom-xxxl-desktop {
		margin-bottom: 8rem !important;
	}
	.padding-bottom-xxxl-desktop {
		padding-bottom: 8rem !important;
	}
	.margin-left-xxxl-desktop {
		margin-left: 8rem !important;
	}
	.padding-left-xxxl-desktop {
		padding-left: 8rem !important;
	}
}
[etc]
```

Reply:
Apologies, this truly is an obnoxiously long comment 😑

Reply:
That's fine, can we resolve it though? Since it's just a demo if the output